src/algebra/lie/basic.lean

```diff
@@ -474,6 +480,33 @@ lie_module.is_trivial.trivial x m
 abbreviation is_lie_abelian (L : Type v) [has_bracket L L] [has_zero L] : Prop :=
 lie_module.is_trivial L L
 
+lemma lie_abelian_of_injection {R : Type u} {L₁ : Type v} {L₂ : Type w}
```

Comment:
We have similar things for `monoid`, `comm_ring`, etc, and there the declarations are put in the `function.{injective|surjective}` namespace.
So `function.injective.is_lie_abelian`, and then you can use it as `h1.is_lie_abelian h2`.

Reply:
Not even out for review and you're helping!

Thanks, done :-)